lib/l10n/app_fr.arb

```
@@ -265,6 +314,68 @@
+  "no_recent_expenses": "Pas de dépenses récentes",
+  "unknown": "Inconnu",
+  "recurring_expense": "Dépense récurrente",
+  "recurring_income": "Revenu récurrent",
+  "recurrence_type": "Type de récurrence",
+  "recurrence_interval": "Intervalle",
+  "recurrence_none": "Aucune",
+  "recurrence_daily": "Quotidienne",
+  "recurrence_weekly": "Hebdomadaire",
+  "recurrence_monthly": "Mensuelle",
+  "recurrence_yearly": "Annuelle",
+  "end_date": "Date de fin",
```

RuboCop (1.73)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)